app/uk/gov/hmrc/gform/validation/ValidationUtil.scala

```diff
@@ -121,15 +121,18 @@ object ValidationUtil {
       case IsAddress(Address(_, _, _, Some(AuthCtx(AuthInfo.ItmpAddress)))) =>
```

Comment:
I know this should be flagged in other ticket, but this logic seem incorrect to be here.
Consequence of this being here is that you **can't edit** prepopulate address component. Ability to edit whatever is populated is what we should be able to do.

I would expect prepopulation logic to be included in `SectionRenderingService.scala`

Reply:
yes, that's the problem I haven't thought about. It won't be in SectionRederingService only. It also needs to be rendered properly in  CYA, robotics and pdf as far as I understand.  
Personally I would rather try added a logic to update Addresses that refer to another Address once the original Address is changed. Not sure how easy it would be though.

Reply:
> rendered properly in CYA, robotics and pdf as far as I understand.

We should get this for free. Address component rendering should not be tied to the source of data (ie. user entered vs prepopulated from itmpAddress). 

> logic to update Addresses that refer to another Address once the original Address is changed

This sounds completely out of the scope of what prepop should be.